app/src/main/java/com/veriff/demo/data/dataSources/sessionToken/VeriffSessionTokenDataSource.kt

```diff
@@ -0,0 +1,64 @@
+package com.veriff.demo.data.dataSources.sessionToken
+
+import com.google.gson.Gson
+import com.veriff.demo.AppStatics
+import com.veriff.demo.BuildConfig
+import com.veriff.demo.data.TokenPayload
+import com.veriff.demo.data.TokenResponse
+import com.veriff.demo.service.AppNetworkService
+import com.veriff.demo.utils.GeneralUtils
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class VeriffSessionTokenDataSource(private val appNetworkService: AppNetworkService,
```

Comment:
@madisp had another solution for this I guess

Reply:
Have added the TokenPayload changes.